src/Raven.Server/Documents/Sharding/Queries/ShardedQueryProcessor.cs

```diff
@@ -306,23 +306,36 @@ public ValueTask HandleIncludes()
             return ValueTask.CompletedTask;
         }
 
-        return HandleIncludesAsync(missing);
+        return HandleMissingIncludesAsync(missing);
     }
 
-    private async ValueTask HandleIncludesAsync(HashSet<string> missing)
+    private async ValueTask HandleMissingIncludesAsync(HashSet<string> missing)
     {
         var includeCommands = new Dictionary<int, ShardedQueryCommand>();
 
-        IDisposable toDispose = null;
-        var missingAsSlice = missing.Select(key =>
+        const string listParameterName = "p0";
+
+        var shards = ShardLocator.GetDocumentIdsByShards(_context, _parent.DatabaseContext, missing);
+
+        var documentQuery = new DocumentQuery<dynamic>(null, null, "@all_docs", false);
+        documentQuery.WhereIn("id()", new List<object>());
+        var query = documentQuery.ToString();
```

Comment:
Why? You need a `load` here, not even a query.
And anyway, this is a static string

Reply:
done